src/sidebars/favourites/favourite-controller.js

```diff
@@ -156,6 +158,19 @@
             });
         }
 
+        // pseudo private
+        _addError(newError) {
+            var err = this.errors, max = err.length;
+            for (var i = 0; i < max; i++) {
```

Comment:
Using a forEach would be less efficient though since we'd have to iterate through all elements regardless of when we found it.
On the other hand, it probably doesn't matter since there should rarely be more than 2 showing at any given time.

I'll actually change it to an object and use the error code as the key, this'll remove the "for" altogether. 
This way it'll also match the observable structure